install.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
```

Comment:
I would actually prefer if this script just went away. Now when the third-party plugins are [officially supported](https://shapeshed.com/vim-packages/) by ``vim >= 8.0``, isn’t it time to just not bother with this script, please? @noelhenson , what do you think?